src/test/lib/DataTest.cpp

```diff
@@ -3280,7 +3280,7 @@ QuicTestConnectAndIdleForDestCidChange(
     }
 }
 
-#define BUFFER_SIZE 10000
+#define BUFFER_SIZE 30000
```

Comment:
This buffer needs to by dynamically allocated on the heap, not the stack because Windows kernel mode can bugcheck.

Reply:
Done! 91f4021db5f2d340a613e505675611cb83c29e18